marketplace-and-integrations/packages/authorized-services.md

```diff
@@ -0,0 +1,776 @@
+  "UseProofKeyForCodeExchange": false,
+  "RequestTokenPath": "/api/oauth.v2.access",
+  "RequestTokenFormat": "FormUrlEncoded",
+  "JsonSerializer": "SystemTextJson",
+  "ClientId": "",
+  "ClientSecret": "",
+  "Scopes": "users:read.email users:read channels:read groups:read mpim:read im:read",
+  "SampleRequest": "/api/conversations.list?limit=50"
+}
+```
+
+### Timely
+
+```json
+{
+  "Alias": "timely",
+  "DisplayName": "Timely",
+  "ApiHost": "https://api.timelyapp.com/1.1",
+  "IdentityHost": "https://api.timelyapp.com/1.1",
+  "TokenHost": "https://api.timelyapp.com/1.1",
+  "RequestIdentityPath": "/oauth/authorize",
+  "AuthorizationUrlRequiresRedirectUrl": true,
+  "RequestTokenPath": "/oauth/token",
+  "RequestTokenFormat": "FormUrlEncoded",
+  "ClientId": "",
+  "ClientSecret": "",
+  "Scopes": "",
+  "SampleRequest": "/1.1/accounts"
+}
+```
+
+### Twitter
+
+```json
+{
+  "Alias": "twitter",
+  "DisplayName": "Twitter",
+  "ApiHost": "https://api.twitter.com",
+  "IdentityHost": "https://twitter.com",
+  "TokenHost": "https://api.twitter.com",
+  "RequestIdentityPath": "/i/oauth2/authorize",
+  "RequestTokenPath": "/2/oauth2/token",
+  "RequestTokenFormat": "FormUrlEncoded",
+  "AuthorizationRequestsRequireRedirectUri": true,
+  "UseProofKeyForCodeExchange": true,
+  "ClientId": "",
+  "ClientSecret": "",
+  "Scopes": "offline.access tweet.read users.read",
+  "SampleRequest": "/2/users/me"
+},
+```
+
+### Yahoo
+
+```json
+{
+  "Alias": "yahoo",
+  "DisplayName": "Yahoo",
+  "ApiHost": "https://api.login.yahoo.com",
+  "IdentityHost": "https://api.login.yahoo.com",
+  "TokenHost": "https://api.login.yahoo.com",
+  "RequestIdentityPath": "/oauth2/request_auth",
+  "AuthorizationUrlRequiresRedirectUrl": true,
+  "UseProofKeyForCodeExchange": false,
+  "RequestTokenPath": "/oauth2/get_token",
+  "RequestTokenFormat": "FormUrlEncoded",
+  "JsonSerializer": "SystemTextJson",
+  "ClientId": "",
+  "ClientSecret": "",
+  "Scopes": "",
+  "SampleRequest": "/openid/v1/userinfo"
+}
+```
+
+### Zendesk
+
+```json
+{
+  "Alias": "zendesk",
+  "DisplayName": "Zendesk",
+  "ApiHost": "https://api.getbase.com",
+  "IdentityHost": "https://api.getbase.com",
+  "TokenHost": "https://api.getbase.com",
+  "RequestIdentityPath": "/oauth2/authorize",
+  "AuthorizationUrlRequiresRedirectUrl": true,
+  "RequestTokenPath": "/oauth2/token",
+  "RequestTokenFormat": "FormUrlEncoded",
+  "ClientId": "",
+  "ClientSecret": "",
+  "Scopes": "read",
+  "SampleRequest": "/v2/contacts"
+}
+```
+
+## Contributing
+
+The Authorized Services package is open-source and we welcome issues, suggestions for improvement or PRs.
+
+You can find the [source code and issue tracker at GitHub](https://github.com/umbraco/Umbraco.AuthorizedServices).
+
+The readme file there contains further information, expanding on the documentation you have read here. This will help anyone interested in understanding how it has been developed and interested in contributing to the solution.
```

Comment:
```suggestion
The readme file there contains further information, expanding on the documentation you have read here. This will help anyone interested in understanding how it has been developed and how to contribute to the solution.
```